README.rst

```diff
@@ -74,6 +76,14 @@ For example:
          author, 'manual'),
     ]
 
+    html_context = {
+        "display_github": True, # Integrate GitHub
+        "github_user": "edx", # Username
+        "github_repo": 'edx-sphinx-theme', # Repo name
+        "github_version": "master", # Version
+        "conf_py_path": "/docs/", # Path in the checkout to the docs root
```

Comment:
Note: I like this better **without** the edX specific comment.  Just noting, because it looked like you added and removed a couple of times.